### PR TITLE
fix: print out "Coming Soon" when there's no downloadable link

### PR DESCRIPTION
**Pull request by ghost**
Ref: https://github.com/nodejs/nodejs.org/pull/4101.

### CODE REVIEW

**Review by targos — DISMISSED** [review]
If we don't put "Coming soon", releasers will not realize when a link is missing.

**Comment by MaledongGit** [comment]
It seems we have to check manually for "Coming Soon...."

**Comment by targos** [comment]
I think something nice would be to additionally print a message to the console. /cc @nodejs/releasers wdyt?

**Comment by MaledongGit** [comment]
Glad to help, here's my draft left here :)

**Comment by ghost** [comment]
@targos : Fix it by consoling in green to notify you, waiting for others' reviews.

**Comment by ghost** [comment]
@targos : Maybe you can remove the "Change Request" icon because I've changed it? XD

File: scripts/release-post.js [hunk 1]
```diff
@@ -153,7 +153,10 @@ function findAuthorLogin (version, section) {
 function urlOrComingSoon (binary) {
   return sendRequest({ url: binary.url, method: 'HEAD' }).then(
     () => `${binary.title}: ${binary.url}`,
-    () => `${binary.title}: *Coming soon*`
+    () => {
+      console.log(`\x1B[32m "${binary.title}" is Coming Soon...\x1B[39m`)
```
Comment (targos) [inline scripts/release-post.js]:
```suggestion
      console.log(`\x1B[32m "${binary.title}" is Coming soon...\x1B[39m`)
```

**Review by targos — APPROVED** [review]

**Pull request merged**